src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -174,38 +176,52 @@ const AugmentedRealityView = ({ sceneNavigator }) => {
 
   // TODO: these data can be updated according to the data coming from the server when the
   //       real 3D models arrive
-  const position = [0, -1, -5];
+  const position = [0, 0, 0];
   const rotation = [0, 0, 0];
-  const scale = [0.02, 0.02, 0.02];
+  const scale = [0.002, 0.002, 0.002];
+
+  ViroARTrackingTargets.createTargets({
+    targetImage: {
+      physicalWidth: 0.2, // real world width in meters
+      source: { uri: object.target },
+      type: 'image'
+    }
+  });
 
   return (
     <ViroARScene dragType="FixedToWorld">
       <ViroAmbientLight color={'#fff'} />
 
-      {!!object.mp3 && !isObjectLoading && (
-        <ViroSound
-          source={{ uri: object.mp3 }}
-          paused={!isStartAnimationAndSound}
-          onFinish={() => setIsStartAnimationAndSound(false)}
-        />
-      )}
+      <ViroARImageMarker
+        onAnchorFound={() => setIsStartAnimationAndSound(true)} // animation and sound file are started after the image is recognised
+        pauseUpdates={true} // prevents the model from continuous jumping
```

Comment:
`pauseUpdates` would be enough to pass `true` here as prop, right?

Reply:
Yes, you're right.